src/app/Login/page.tsx

```diff
@@ -0,0 +1,91 @@
+"use client"
+import { useState } from "react"
+import { Container, Typography, TextField, Button, Snackbar } from "@mui/material"
+import { useNavigate } from "react-router-dom"
+import './Login.css'
+
+function Login() {
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [error, setError] = useState<string | null>(null)
+    const [openSnackbar, setOpenSnackbar] = useState<boolean>(false)
+    //const navigate = useNavigate()
```

Comment:
```suggestion
    import { useRouter } from "next/router"
    /* ... */
    const router = useRouter()
```
Next.js는 react-router-dom이 아니라 자체 라우터를 사용하므로 `useRouter`를 사용하시면 됩니다! useNavigate와 사용법은 비슷할 텐데, 자세한 것은 [공식문서](https://nextjs.org/docs/app/api-reference/functions/use-router) 참고하세용